java/spacewalk-java.spec

```diff
@@ -517,9 +517,7 @@ install -m 644 conf/rhn_java.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-def
 install -m 644 conf/rhn_java_sso.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults
 
 # Adjust product tree tag
-%if 0%{?sle_version} && !0%{?is_opensuse}
```

Comment:
Should we maybe consider keeping the if/else and just comment the sed, so it's easier to re-enable next time?

Reply:
I am not sure if an it is allowed to have an empty block.
But this is the reason why I tried to invert the original %if, now we can just add %else and put the statement back.